contracts/seed/Seed.sol

```diff
@@ -177,8 +173,8 @@ contract Seed {
         fundingToken      = IERC20(_tokens[1]);
         fee               = _fee;
 
-        seedAmountRequired = (hardCap*PCT_BASE) / _price;
-        feeAmountRequired  = (seedAmountRequired*_fee) / 100;
+        seedAmountRequired = (hardCap*PRECISION) / _price;
+        feeAmountRequired  = (seedAmountRequired*_fee) / PRECISION;
```

Comment:
the fee is in the order of `10**16`. That is 1% = `10**16` or `toWei('0.01')`. 
So for the calculation `(seedAmountRequired * fee) / 100`, now it becomes `(seedAmountRequired * fee) / (100 * (10**16)`, it can also be written as `(seedAmountRequired * fee) / 10**18` which is `PRECISION`.
This was done to have only one variable for precision related things. Also, fee is in `10**16` denomination so that `PRECISION` can help with percent related calculation.